build.gradle

```diff
@@ -73,17 +94,35 @@ dependencies {
 
     compileOnly('org.springframework.boot:spring-boot-configuration-processor')
 
-	runtime('org.springframework.boot:spring-boot-devtools')
+    runtime('org.springframework.boot:spring-boot-devtools')
 
-    testCompile("junit:junit")
-	testCompile('org.springframework.boot:spring-boot-starter-test')
-	testCompile('org.springframework.restdocs:spring-restdocs-mockmvc')
+    testCompile('junit:junit')
+    testCompile('org.springframework.boot:spring-boot-starter-test')
+    testCompile('org.springframework.restdocs:spring-restdocs-mockmvc')
     testCompile("com.winterbe:expekt:$expekt_version")
     testCompile("org.jetbrains.spek:spek-api:$spek_version")
     testCompile("org.junit.jupiter:junit-jupiter-api:$junit_version")
     testCompile("com.natpryce:hamkrest:$hamkrest_version")
     testCompile("com.github.sbrannen:spring-test-junit5:1.0.0")
 
     testRuntime("org.junit.jupiter:junit-jupiter-engine:$junit_version")
+    testRuntime("org.junit.platform:junit-platform-console:$junit_gradle_version")
     testRuntime("org.jetbrains.spek:spek-junit-platform-engine:$spek_version")
 }
+
+// https://stackoverflow.com/questions/45297339/junit-5-under-gradle-with-multiple-source-sets
+task 'test-e2e'(
+    type: JavaExec,
+    description: 'Runs the e2e tests.',
+    group: 'Verification'
+) {
+    dependsOn testE2eClasses
+
+    outputs.upToDateWhen { false }
+    classpath = sourceSets.'test-e2e'.runtimeClasspath
+
+    main = 'org.junit.platform.console.ConsoleLauncher'
+    args = ['--scan-class-path',
+            sourceSets.'test-e2e'.output.getClassesDirs().asPath,
+            '--reports-dir', "${buildDir}/test-results/test-e2e"]
+}
```

Comment:
kludgy... but can't find a better way to get junit 5 to run tests in a different sourceset than 'test' sourceset...  hoping this gets fixed eventually